### ISSUE TITLE
Waze don't work after compile 

### ISSUE THREAD

**Issue by chuppito** [chuppito/Waze#1]
Hello everyone, I opened this post to ask for your help. I hope you can help me.

I modified waze after passing a first security. Now when I modify it and sign it with my own signature, some functions are blocked (like android auto and Spotify for example).
I guess he checks if it's the original signature.
Can you help me find out how to steal this signature?
I put the file smali download and the code of the most propitable (I hesitate anyway with another) on github.
Thank you all

@coolnickname
@sigalor
@ccaapton
@topjohnwu 
@slashmax

**Comment by ccaapton** [comment]
If an application is so dedicated in checking its own signature, there is very little you could hack.

**Comment by chuppito** [comment]
Yes I do not try to reproduce the signature, just make him pass the control for it to work as I have seen on other applications (whatsapp, yik yak).
